sigmastate/src/main/scala/sigmastate/serialization/GroupElementSerializer.scala

```diff
@@ -21,12 +22,12 @@ object GroupElementSerializer extends SigmaSerializer[EcPointType, EcPointType]
   private lazy val identityPointEncoding = Array.fill(encodingSize)(0: Byte)
 
   override def serialize(point: EcPointType, w: SigmaByteWriter): Unit = {
-    val bytes = if (point.isInfinity) {
+    val bytes = if (CryptoFacade.isInfinityPoint(point)) {
```

Comment:
Better import CryptoFacade functions in the beginning of the method to have short names after, readability of the new version is poor

Reply:
It is matter of taste. This way is more explicit. Anyway, this is not significant and I would better leave it as is. (I already expect enough troubles with merging)